css/style.css

```diff
@@ -0,0 +1,78 @@
+* {
+	margin: 0;
+    box-sizing: border-box;
+}
+
+body {
+  margin: 0px;
+  background-color: #83A298;
+  padding: 0px;
+  min-height: 100%;
+  width: 100%;
+}
+
+/* Style query form */
+.queryform {
+    overflow: hidden;
+    background-color: #BCCDC7;
+    margin: auto;
+    padding: 10px;
+    margin: 20px 20px 0px 20px;;
+}
+
+/* Style the past search links */
+.history a {
+    float: left;
+    display: block;
+    color: #f2f2f2;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+}
+
+/* Change color on hover */
+.history a:hover {
+    background-color: #ddd;
+    color: black;
+}
```

Comment:
These `.history` rules don't seem to be in use.

Reply:
These are for a later feature - they will be utilized later.